content/400-reference/300-database-reference/02-connection-urls.mdx

```diff
@@ -75,6 +76,15 @@ datasource db {
 }
 ```
 
+### CockroachDB (Preview)
+
+```prisma file=schema.prisma
+datasource db {
+  provider = "cockroachdb"
+  url      = "postgresql://janedoe:mypassword@localhost:26257/mydb?schema=public"
+}
```

Comment:
Does this usually get a generator block for the preview feature as long as in preview?

Reply:
I've added one in